libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Solo.h

```diff
@@ -18,6 +18,7 @@ class AP_BattMonitor_SMBus_Solo : public AP_BattMonitor_SMBus
     void timer(void) override;
 
     uint8_t _button_press_count;
+    bool _use_extended = false;
```

Comment:
```suggestion
    bool _use_extended;
```
Don't need to initialize false.